### DISCUSSION TITLE
Revamp

### DISCUSSION THREAD

**Discussion by Ctrl-sza18** [fast4x/RiMusic#3777] [Ideas]
Hey I was thinking about the player don't get me wrong it amazing it has customizable option but I was thinking you can make them more and change the player to another revamped one if you want to make it not similar to the picture I have no problem

If you have time to do it you can prob release it on the next update or two
![5ea23a39e6c069cb498b5c78cec306d6.jpg](https://github.com/user-attachments/assets/fe38b8c8-59a4-4b70-9948-3cb46e67e730)

![1429f7d58dde1fdc9694bdd8f78226b7.jpg](https://github.com/user-attachments/assets/ef75e5f4-2bc7-4609-8fae-c53ea74567a9)

![a4550d95d1395a58e79029075dfc6654.jpg](https://github.com/user-attachments/assets/0b49ff90-51e0-45cb-9464-5531a4993218)

![f5210870dd9e25e61a3bf507d3c5b781.jpg](https://github.com/user-attachments/assets/971c6e13-976f-47bb-ab82-e99574cc2339)

![9fb15e7fe6bc9eb5dfa1e06d054f6a43.jpg](https://github.com/user-attachments/assets/a4525b4c-0c38-4305-8f8f-2039d7d928ac)

**Comment by fast4x** [comment]
Too generic, RiMusic have a lotof  customizations, try all please.
